src/render.rs

```diff
@@ -177,9 +177,7 @@ impl Renderer {
         )
         .unwrap();
         let cursor = cursor_theme
-            .get_cursor("default")
-            .expect("Could not load cursor, check XCURSOR_THEME")
-            .clone();
+            .get_cursor("default").ok_or(String::from("Could not load cursor, check XCURSOR_THEME"))?;
```

Comment:
You should just return if `get_cursor` is `None`, not of create a `String` that you then ignore.  If you want to have an error message, use a log macro like `warn!`.